app/src/main/java/com/onfido/android/app/sample/MainActivity.java

```diff
@@ -1,21 +1,32 @@
 package com.onfido.android.app.sample;
 
+import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Toast;
 
+import com.androidnetworking.AndroidNetworking;
+import com.androidnetworking.error.ANError;
+import com.androidnetworking.interfaces.JSONObjectRequestListener;
 import com.onfido.android.sdk.capture.ExitCode;
 import com.onfido.android.sdk.capture.Onfido;
+import com.onfido.android.sdk.capture.OnfidoConfig;
 import com.onfido.android.sdk.capture.OnfidoFactory;
 import com.onfido.android.sdk.capture.ui.options.FlowStep;
 import com.onfido.android.sdk.capture.ui.options.MessageScreenStep;
 import com.onfido.android.sdk.capture.upload.Captures;
+import com.onfido.android.sdk.capture.utils.OnfidoApiUtil;
 import com.onfido.api.client.data.Applicant;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 public class MainActivity extends BaseActivity {
 
     private Onfido client;
+    private Activity activity = this;
```

Comment:
unusual pattern. Inside of callbacks the common pattern is to do `MainActivity.this`.

Reply:
👍